packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -642,5 +642,8 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
     legal: {
       privacyStatement: `${ELASTIC_WEBSITE_URL}legal/privacy-statement`,
     },
+    kibanaUpgradeSavedObjects: {
+      resolveMigrationFailures: `${KIBANA_DOCS}resolve-migrations-failures.html`,
```

Comment:
Is that supposed to point to [docs/setup/upgrade/resolving-migration-failures.asciidoc](https://github.com/elastic/kibana/pull/130352/files#diff-e53d56b82e5ce58f55b4f371ced6bec7bd5df2fe364ad73ac3977df8e93ba03f) (if so, the path is wrong I believe)? Or is it another page?

Reply:
The webpage path to those docs is [https://www.elastic.co/guide/en/kibana/current/resolve-migrations-failures.html](https://www.elastic.co/guide/en/kibana/current/resolve-migrations-failures.html) and `${KIBANA_DOCS}` -> `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/`, so the path defines the path to the webpage and not to the raw `asciidoc` in github.